pkg/handler/nfts.go

```diff
@@ -28,6 +28,19 @@ func (h *Handler) CreateNFTCollection(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 		return
 	}
+
+	checkExitsNFT, err := h.entities.CheckExistNftCollection(req.Address)
+	// err == nil means exist
```

Comment:
if err = nil then you duplicate the handle error
what if error is not nil, then ?